tests/test_database_update.py

```diff
@@ -73,33 +73,34 @@ def test_database_add_and_update_s3():
 
 
 @pytest.mark.integration
-def test_database_update_local():
+def test_database_update_local_list():
+    base_test_database_update_local(test_with_prefix=False)
+
+
+@pytest.mark.integration
+def test_database_update_local_prefix():
+    base_test_database_update_local(test_with_prefix=True)
+
+
+def base_test_database_update_local(test_with_prefix):
```

Comment:
Nit: this looks like a good use of `@pytest.paramaterize`: https://docs.pytest.org/en/6.2.x/parametrize.html